docs/the_black_code_style/future_style.md

```diff
@@ -62,3 +62,52 @@ plain strings. User-made splits are respected when they do not exceed the line l
 limit. Line continuation backslashes are converted into parenthesized strings.
 Unnecessary parentheses are stripped. The stability and status of this feature is
 tracked in [this issue](https://github.com/psf/black/issues/2188).
+
+### Improved line breaks
+
+For assignment expressions, it now prefers to split and wrap the right side of the
+assignment instead of left side. For example:
```

Comment:
```suggestion
For assignment expressions, _Black_ now prefers to split and wrap the right side of the
assignment instead of left side. For example:
```